sdk/include/reactos/helpofhtml.h

```diff
@@ -0,0 +1,178 @@
+/*
+ * PROJECT:     ReactOS HelpOfHtml (HOH)
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     ReactOS-specific help system
+ * COPYRIGHT:   Copyright (c) 2025 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#pragma once
+
+#ifdef ENABLE_HELP
+
+#include <winreg.h>
+#include <shellapi.h>
+#include <shlwapi.h>
+#include <strsafe.h>
+
+// Does registry key exist?
```

Comment:
Not sure if we need this 'obvious comment'...

Reply:
```suggestion
```